infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerResolverTest.java

```diff
@@ -263,4 +265,10 @@ private Ingress createIngress(
         .endStatus()
         .build();
   }
+
+  private Map<String, String> defaultAttributeAnd(String key, String value) {
```

Comment:
It is not so important but still `defaultAttributeAnd` -> `defaultAttributesAnd`

Reply:
Since there is only one default attribute I would leave it as is

Reply:
But you use map as default attributes, later someone may just put new entry in `ATTRIBUTES_MAP` and do you even notice this private method. Up to you since it is a private method of test class.